server/cmd/mmctl/commands/user_test.go

```diff
@@ -2748,6 +2748,7 @@ func (s *MmctlUnitTestSuite) TestPromoteGuestToUserCmd() {
 
 		err := promoteGuestToUserCmdF(s.client, nil, []string{emailArg})
 		s.Require().NoError(err)
```

Comment:
This seems off

Reply:
deleted line 2750, but I wanna know is there is any docs regarding to the testing?